.packit.yaml

```
@@ -30,8 +30,8 @@ jobs:
   - job: copr_build
     trigger: pull_request
     targets:
-      centos-stream-8:
-        additional_modules: "foreman:el8,nodejs:12"
+      rhel-8:
+        additional_modules: "foreman:el8,nodejs:14"
```

```suggestion
        additional_modules: "foreman:el8"
```

We don't use nodejs (and if we'd did, we should use `foreman-devel:el8` module instead)